chart/kubeapps/templates/_helpers.tpl

```diff
@@ -183,15 +183,22 @@ a defined apiServiceURL.
         {{- fail "At least one cluster must be defined." }}
     {{- end }}
     {{- range .Values.clusters }}
-        {{- if eq (.apiServiceURL | toString) "<nil>" }}
-            {{- if eq $kubeappsCluster "" }}
+        {{- if eq $kubeappsCluster "" }}
+            {{- if or (eq (.isKubeappsCluster | toString) "true") ( eq (.apiServiceURL | toString) "<nil>") }}
```

Comment:
Oops, indeed I didn't. I assumed it was better to use the `toString` value to cover both `isKubeappsCluster: "false"` and `isKubeappsCluster: false` user inputs. If we don't, "false" will be interpreted as true, afaik.

I'm a little concerned about it, since we don't any json schema to validate our yaml against, we cannot assure the received values are syntactically correct before starting the semantic analysis.

But, yep, for the sake of consistency with the rest of the values.yaml, I'm changing to the proposed suggestion.